src/test/scala/uk/gov/hmrc/api/client/HttpClient.scala

```diff
@@ -16,26 +16,34 @@
 
 package uk.gov.hmrc.api.client
 
-import akka.actor.ActorSystem
+import org.apache.pekko.actor.typed.{ActorSystem, Behavior}
+import org.apache.pekko.actor.typed.scaladsl.Behaviors
 import play.api.libs.json.{JsValue, Json}
 import play.api.libs.ws.DefaultBodyWritables._
 import play.api.libs.ws.JsonBodyWritables.writeableOf_JsValue
-import play.api.libs.ws.StandaloneWSRequest
+import play.api.libs.ws.{StandaloneWSRequest, StandaloneWSResponse}
 import play.api.libs.ws.ahc.StandaloneAhcWSClient
 import uk.gov.hmrc.api.conf.ZapConfiguration._
 
 import scala.concurrent.{ExecutionContext, Future}
 
+object MyActor {
+  def apply(): Behavior[String] = Behaviors.receive { (context, message) =>
+    context.log.info(s"Received message: $message")
+    Behaviors.same
+  }
+}
+
 trait HttpClient {
 
-  implicit val actorSystem: ActorSystem = ActorSystem()
-  implicit val ec: ExecutionContext     = ExecutionContext.global
+  implicit val actorSystem: ActorSystem[String] = ActorSystem(MyActor(), "myActorSystem")
```

Comment:
implicit lazy val actorSystem: ActorSystem        = ActorSystem()

This should be enough.